test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -17,19 +17,19 @@ public DownloadFilesController(IHostingEnvironment hostingEnvironment)
             _hostingEnvironment = hostingEnvironment;
         }
 
-        public IActionResult DowloadFromDisk()
+        public IActionResult DownloadFromDisk()
```

Comment:
Hmm, I wonder - if these are being changed here, where else do they need to change? Presumably these methods are called from somewhere and the callsites need to be updated, or perhaps these methods aren't called at all and can be deleted?

@dougbu @pranavkm any idea?